primeqa/ir/dense/colbert_top/colbert/modeling/factory.py

```diff
@@ -76,13 +68,6 @@ def get_colbert_from_pretrained(name, colbert_config):
             colbert.load_state_dict(name)
         else:
             colbert = HF_ColBERT_Roberta.from_pretrained(name, colbert_config)
-    elif model_type == 'custom':
```

Comment:
@franzmpub : With this flag removed, wow are we supporting instantiating colBERT with a custom config now ?

Reply:
@jdpsen (FYI @riyazbhat ): `model_type` will be the corresponding model type (e.g. `roberta` for WatBERT). 
The `custom` type was just inherited from the testing phase

Reply:
@jdpsen (FYI @riyazbhat ): `model_type` will be the corresponding model type (e.g. `roberta` for WatBERT). 
The `custom` type was just inherited from the testing phase